roles/cluster_logging/templates/kibana-index-provisioning-configmap.yml.j2

```diff
@@ -7,5 +7,7 @@ metadata:
     heritage: MetalK8s
     component: index-provisioning
 data:
+  index.pattern: {{ kibana_index_pattern }}
```

Comment:
Not sure it makes sense setting things through the `ConfigMap`. Just set them in `env` in the `Job` template.